tests/toplevel_unit.c

```diff
@@ -1,27 +1,26 @@
 #include <assert.h>
+#include <roaring/misc/configreport.h>
```

Comment:
Yes. I made not change to toplevel_unit.c per se, but I got annoyed at the fact that it was indented in an irregular fashion. I would otherwise have used a different commit for such a change, but you can trust me that I did not remove any test.